code/datums/gamemode/objectives/ninja/stealsake.dm

```diff
@@ -0,0 +1,13 @@
+/datum/objective/stealsake
+    name = "\[Ninja\] Steal a bottle of sake"
+    explanation_text = "You have gone without a drink for far too long. Steal a bottle of sake, the only beverage worthy of your consumption."
+
+/datum/objective/stealsake/IsFulfilled()
+    if(..())
+        return TRUE
+    
+    if (owner && owner.current)
+        for(var/obj/O in get_contents_in_object(owner.current))
+            if (istype(O, /obj/item/weapon/reagent_containers/food/drinks/bottle/sake))
```

Comment:
This has the bizarre result that an empty bottle of sake succeeds but a beaker with 100u sake fails.

Check reagents.has_reagent(SAKE,**amount**)

Reply:
That's intentional. Though perhaps the jectie should be reworded to reflect this ("Steal a bottle of Uchuujin Junmai Ginjo Sake" or something). If you think it's a dumb idea to target the specific bottle I can change it to reagents too.

Reply:
It can be up for discussion. I feel like it makes more sense to steal a target amount of sake than to be the proud owner of a bottle though.

Reply:
To be fair it is a nice bottle.

Reply:
I think it'd be cooler to steal either the bottle (full, empty) or any container with sake in it.
Or have sake in your system at roundend. Stolen sake imbibed? There's wiggle room.